app/assets/javascripts/discourse/scripts/async-stylesheets.js

```diff
@@ -0,0 +1,22 @@
+// This script is inlined in `_discourse_stylesheet.html.erb
+const links = document.getElementsByClassName("async-css-loading");
+
+const processEvent = function (element) {
+  delete element.dataset["media"];
```

Comment:
```suggestion
element.removeAttribute("media");
```

Is this correct? We have link
`<link class='async-css-loading' href="#{href}" media="print" rel="stylesheet" type="text/css" data-media="#{media}" data-target="#{target}" #{data_theme_id} #{data_theme_name}/>`

I think we want to remove `media="print"` attribute and not `data-media="#{media}"`.

Reply:
Yup this is correct. `data-media` is just for us to store the intended `media` attribute. Once the event has been processed, there is no use for it anymore.